mybackend/models/User.js

```diff
@@ -17,13 +19,55 @@ const userSchema = new mongoose.Schema({
   twitter: String,
   linkedin: String,
   bio: String,
-  completedTasks: [{ taskId: Number, trainingId: mongoose.Schema.Types.ObjectId }],
-  
-  approvedSubmissions: [{
-    competitionId: { type: mongoose.Schema.Types.ObjectId, ref: 'Competition' },
-    submissionType: { type: String, enum: ['Feature', 'Optimization', 'Bug'] },
-    payout: Number
-  }]
+  completedTasks: [
+    { taskId: Number, trainingId: mongoose.Schema.Types.ObjectId },
+  ],
+
+  approvedSubmissions: [
+    {
+      competitionId: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'Competition',
+      },
+      submissionType: {
+        type: String,
+        enum: ['Feature', 'Optimization', 'Bug'],
+      },
+      payout: Number,
+    },
+  ],
+});
+
+// add indexes
+userSchema.index({ xp: -1 });
+userSchema.index({ username: 1 });
```

Comment:
change this to this `userSchema.index({ xp: -1, username: 1 });`

Reply:
done